### DISCUSSION TITLE
Welcome to Gemini-API Discussions!

### DISCUSSION THREAD

**Discussion by HanaokaYuzu** [HanaokaYuzu/Gemini-API#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pookie-ux** [comment]
Is there a way to set maximum tokens??

**Reply by HanaokaYuzu** [reply]
I'm afraid not. Google has never provided this option to users.

**Comment by 0wwafa** [comment]
in the real gemini api there is... I think this project should use https://aistudio.google.com/app/prompts/new_chat

**Reply by HanaokaYuzu** [reply]
ref #19

**Comment by dingliangyi** [comment]
Can support pdf files

**Reply by HanaokaYuzu** [reply]
No. Gemini itself doesn't provide pdf support. 

<img width="918" alt="image" src="https://github.com/user-attachments/assets/3d623f78-d16b-4e90-b3b7-14291bac880d">
